src/Normalizer/ContentEntityNormalizer.php

```diff
@@ -170,7 +169,7 @@ public function normalize($entity, $format = NULL, array $context = array()) {
    *
    * @throws \Symfony\Component\Serializer\Exception\UnexpectedValueException
    */
-  public function denormalize($data, $class, $format = NULL, array $context = array()) {
+  public function denormalize(array $data, string $class, string $format = NULL, array $context = array()) {
```

Comment:
Type hinting on primitive datatypes is not supported in PHP 5
http://php.net/manual/en/functions.arguments.php#functions.arguments.type-declaration